tests/test_calculations.py

```diff
@@ -531,3 +531,26 @@ def test_fourteen_day_avg_ratio(self):
         result = fourteen_day_avg_ratio(df['some_number'])
 
         assert_series_equal(result, df['expected'], check_names=False)
+
+    def test_inf_values(self):
+        measurements = 25
+        districts = 4
+        district_values = [.3, .7, 10, 25]
+        input_df = self._build_district_input(measurements=measurements, districts=districts,
+                                              district_values=district_values)
+
+        hospitalizations = impute_hospitalization_percentages(
+            pd.DataFrame({'date': [datetime(2020, 10, 3)], 'percentages': [0.13]}), input_df['date'])
+        input_df["delta.tested"].replace({10:0}, inplace=True)
```

Comment:
This line probably deserves a comment that you divide by zero and thus normaly expect inf's

Reply:
Thanks Marco! All comments are valid. I will merge this request and then will create another pull request for these comments.